Examples/Python/tests/test_examples.py

```diff
@@ -918,17 +916,17 @@ def test_digitization_example(trk_geo, tmp_path, assert_root_hash):
 
     assert len(list(csv_dir.iterdir())) == 3 * s.config.events
     assert all(f.stat().st_size > 50 for f in csv_dir.iterdir())
-    for tn, nev in (
-        (8, 407),
-        (9, 0),
-        (12, 11),
-        (13, 375),
-        (14, 2),
-        (16, 25),
-        (17, 146),
-        (18, 9),
+    assert_entries(root_file, "vol9", 0)
+    for tn in (
+        8,
+        12,
+        13,
+        14,
+        16,
+        17,
+        18,
     ):
```

Comment:
nitpick should we collapse this on one line now?
```suggestion
    for tn in (8, 12, 13, 14, 16, 17, 18):
```

Reply:
Sure